libs/standard-tests/langchain_standard_tests/unit_tests/chat_models.py

```diff
@@ -18,81 +21,109 @@ def my_adder_tool(a: int, b: int) -> int:
     return a + b
 
 
-class ChatModelUnitTests(ABC):
+class ChatModelTests(ABC):
+    @property
     @abstractmethod
-    @pytest.fixture
     def chat_model_class(self) -> Type[BaseChatModel]:
         ...
 
-    @pytest.fixture
+    @property
     def chat_model_params(self) -> dict:
```

Comment:
do these need to be fixtures? I'm getting `fixture 'chat_model_class' not found` on `test_stop_sequence`.